share/spice/keybase/content.handlebars

```diff
@@ -0,0 +1,25 @@
+<div class="keybase--basic">
+  <div class="keybase--image--wrap">
+    {{#if pictures.primary.url}}
+    <img src="{{imageProxy pictures.primary.url}}" class="circle keybase--image">
+    {{else}}
+    <img src="{{imageProxy 'https://keybase.io/images/no_photo.png'}}" class="circle keybase--image">
+    {{/if}}
+  </div>
+  <div class="keybase--misc">
+    <h1 class="zci__header">{{profile.full_name}}</h1>
```

Comment:
Should this be a hyperlink? Like the [twitter](https://github.com/duckduckgo/zeroclickinfo-spice/blob/6a878b223c89a6f7b8de556bf7803030f6a9e8a7/share/spice/twitter/content.handlebars#L8) IA? @jagtalon